plugin/core/protocol.py

```diff
@@ -57,6 +57,14 @@ class CompletionItemKind(object):
     Reference = 18
 
 
+class DocumentHighlightKind(object):
+    Unknown = 0
+    Text = 1
+    Read = 2
+    Write = 3
+    Count = 4
```

Comment:
I had to double check, but `Count` is a clever definition not in the spec?
Prefer using an Enum here if possible and `len(DocumentHighlightKind)` for count.

Reply:
We can't use enums? we're on Python 3.3. Not entirely sure what you mean here. I've removed the `Count` class member and replaced it with a `4` in the `x in range(0, 4)`.